blocks/quote/quote.js

```diff
@@ -0,0 +1,38 @@
+import { createOptimizedPicture, decorateIcons } from '../../scripts/aem.js';
+import { span } from '../../scripts/dom-helpers.js';
+
+export default function decorate(block) {
+  /* change to ul, li */
+  const ul = document.createElement('ul');
+  [...block.children].forEach((row) => {
+    const li = document.createElement('li');
+    while (row.firstElementChild) li.append(row.firstElementChild);
+    [...li.children].forEach((divElement) => {
+      if (
+        divElement.children.length === 1
+        && divElement.querySelector('picture')
+      ) {
+        divElement.className = 'cards-card-image';
+      } else divElement.className = 'cards-card-body';
+    });
+    ul.append(li);
+  });
+  ul.querySelectorAll('img').forEach((img) => img
```

Comment:
Why is this needed?

Reply:
I'm not sure, I just noticed it was also used in other places.

Should I remove it?